### PR TITLE
Minor consistency changes

### PR DESCRIPTION
**Pull request by Stevoisiak**
Mostly small changes, like capitalization and spelling

### CODE REVIEW

File: Source/Core/Core/HW/DSP.cpp [hunk 1]
```diff
@@ -665,8 +665,8 @@ u8 ReadARAM(u32 _iAddress)
 
 void WriteARAM(u8 value, u32 _uAddress)
 {
-	//NOTICE_LOG(DSPINTERFACE, "WriteARAM 0x%08x", _uAddress);
-	//TODO: verify this on WII
+	// NOTICE_LOG(DSPINTERFACE, "WriteARAM 0x%08x", _uAddress);
+	// TODO: verify this on WII
```
Comment (Buddybenj) [inline Source/Core/Core/HW/DSP.cpp]: Wii shouldn't be all capitalized here (I think).

File: Source/Core/VideoCommon/DriverDetails.h [hunk 2]
```diff
@@ -97,10 +97,10 @@ namespace DriverDetails
 		// Started Version: ?
 		// Ended Version: 13.9 working for me (neobrain).
 		// Affected OS: Linux
-		// Pinned memory is disabled for index buffer as the amd driver (the only one with pinned memory support) seems
+		// Pinned memory is disabled for index buffer as the AMD driver (the only one with pinned memory support) seems
 		// to be broken. We just get flickering/black rendering when using pinned memory here -- degasus - 2013/08/20
 		// This bug only happens when paired with base_vertex.
-		// Please see issue #6105 on google code. Let's hope buffer storage solves this issues.
+		// Please see issue #6105 on Google Code. Let's hope buffer storage solves this issues.
```
Comment (Buddybenj) [inline Source/Core/VideoCommon/DriverDetails.h]: Issue 6105 has been fixed.

File: Source/DSPSpy/main_spy.cpp [hunk 3]
```diff
@@ -353,7 +353,7 @@ void handle_dsp_mail(void)
 		}
 		else if (mail == 0x8888dead)
 		{
-			// Send memory dump (dsp dram from someone's cube?)
+			// Send memory dump (DSP dram from someone's cube?)
```
Comment (Buddybenj) [inline Source/DSPSpy/main_spy.cpp]: DRAM is all capitalized.

Comment (Buddybenj) [inline Source/DSPSpy/main_spy.cpp]: Also you should change it to "someone's GameCube?"

File: Source/Core/VideoCommon/DriverDetails.h (hunk 2 above)
Comment (Stevoisiak) [inline Source/Core/VideoCommon/DriverDetails.h]: What change should I make to signify that?

Comment (Sonicadvance1) [inline Source/Core/VideoCommon/DriverDetails.h]: It's just a note saying what issue it was reported in for the developer if they want more information.
The few lines above state which video driver version fixed the issue.

**Pull request merged**